src/System/Hapistrano/Config.hs

```diff
@@ -1,3 +1,14 @@
+-- |
+-- Module      :  System.Config
+-- Copyright   :  © 2015-Present Stack Builders
+-- License     :  MIT
+--
+-- Maintainer  :  Juan Paucar <jpaucar@stackbuilders.com>
```

Comment:
Was this moved here from somewhere else? If needed, can it be updated (the module name above is incorrect and I'm guessing the maintainer email should be something else)?

Reply:
Yeah the module name is wrong, thanks for the heads-up. I didn't want to change the maintainer; if @juanpaucar is no longer the maintainer, who should be there?

Reply:
I don't know, maybe @CristhianMotoche knows.

Reply:
Hey @DavidMazarro @jpvillaisaza I'm the maintainer at the moment. You can change the name. Thanks.